tests/test_subtractTask.py

```diff
@@ -715,6 +715,76 @@ def _compare_apCorrMaps(self, a, b):
             self.assertFloatsAlmostEqual(
                 value.getCoefficients(), value2.getCoefficients(), rtol=0.0)
 
+    def test_fake_mask_plane_propagation(self):
+        """Test that we have the mask planes related to fakes in diffim images.
+        This is testing method called updateMasks
+        """
+        xSize = 200
+        ySize = 200
+        science, sources = makeTestImage(psfSize=2.4, xSize=xSize, ySize=ySize)
+        science_fake_img, science_fake_sources = makeTestImage(
+            psfSize=2.4, xSize=xSize, ySize=ySize, seed=7, nSrc=2, noiseLevel=0.25, fluxRange=1
+        )
+        template, _ = makeTestImage(psfSize=2.4, xSize=xSize, ySize=ySize, doApplyCalibration=True)
+        tmplt_fake_img, tmplt_fake_sources = makeTestImage(
+            psfSize=2.4, xSize=xSize, ySize=ySize, seed=9, nSrc=2, noiseLevel=0.25, fluxRange=1
+        )
+        # created fakes and added them to the images
+        science.image.array += science_fake_img.image.array
+        template.image.array += tmplt_fake_img.image.array
+
+        # TODO: update to INJECTED names when source injection gets refactored
+        # adding mask planes to both science and template images
+        science.mask.addMaskPlane("FAKE")
+        template.mask.addMaskPlane("FAKE")
+
+        science_mask_planes = science.mask.getMaskPlaneDict()
```

Comment:
You could use `science_fake_mask = science.mask.getMaskPlane('FAKE')` here, and then just use `science_fake_mask` instead of `science_mask_planes['FAKE']` later. Same for the template below.

Or use `science_fake_mask = science.mask.getPlaneBitMask("FAKE")` if you use my suggestion below.